docs/references/ic-interface-spec.md

```diff
@@ -3049,17 +3049,24 @@ The response from the remote server must not exceed `2MB`. Moreover, the total s
 
 Cycles to pay for the call must be explicitly transferred with the call, i.e., they are not automatically deducted from the caller's balance implicitly (e.g., as for inter-canister calls). The upfront cycles cost covers a call that maxes out the resource usage during processing, for example, hitting the `2MB` limit on the size of the response. The unused cycles are then refunded to the caller.
 
-The method may return an error of the `flexible_http_request_err` type. The error includes an error message, and may also include a variant describing the error cause:
+The method may return an error of the `flexible_http_request_err` type. The error includes an error message, and may also include a variant describing the error cause. The causes may include:
+- an optional "global" error preventing the successful response from being created:
 
-- `timeout`, meaning that less than `min_responses` from the nodes have been collected before some system-defined timeout. The value is accompanied by a vector of available resource reports at the point when the timeout went off.
+    - `timeout`, meaning that less than `min_responses` from the nodes have been collected before some system-defined timeout. The value is accompanied by a vector of available resource reports at the point when the timeout went off.
 
-- `invalid_parameters`, indicating that the parameters to the call violated one of the conditions.
+    - `invalid_parameters`, indicating that the parameters to the call violated one of the conditions.
```

Comment:
Why is this a local error? I'd expect that invalid input parameters will be a global error since every node would perform the same validation.

Reply:
I think you misread something here, it's under global errors?